src/plugins/notebook/res/templates/entry.html

```diff
@@ -11,25 +9,26 @@
             <div class="c-ne__text c-input-inline"
                  contenteditable="true"
                  ref="contenteditable"
-                 v-on:blur="textBlur($event, entry.id)"
-                 v-on:focus="textFocus($event, entry.id)"
-                 v-bind:key="entry.id"
+                 @blur="textBlur($event, entry.id)"
+                 @focus="textFocus($event, entry.id)"
                  v-html="entry.text">
             </div>
             <div class="c-ne__embeds">
                 <notebook-embed
-                    v-for="(embed, index) in entry.embeds"
-                    :key="index"
+                    v-for="embed in entry.embeds"
+                    :key="embed.id"
                     :embed="embed"
-                    :entry="entry"
-                ></notebook-embed>
+                    :objectPath="[{identifier: embed.type}]"
```

Comment:
@dtailor90 @larkin We need to resolve this issue before this PR can be merged. It can't be merged as-is, because following embedded links results in invalid navigation state, even though at first glance it appears to have worked.

The embedded object needs the full object path, and the only way I can think of getting that is for it to be provided on drag-and-drop. This would require a minor change everywhere else that the domain object is being received from a drag and drop event. This would be a quick change, and I'd be happy to make it.

Reply:
@akhenry if you want to add object path on drop event with the domainObject, I can modify the code here to use that.